src/main/java/de/mediathekview/mserver/base/utils/UrlUtils.java

```diff
@@ -52,4 +52,28 @@ public static String changeOrAddParameter(final String aUrl, final String aParam
     newUrlBuilder.append(String.format(PARAMETER_PATTERN, aParameter, aValue));
     return newUrlBuilder.toString();
   }
+  
+  /**
+   * returns the base of the url
+   * example: 
+   * https://www.myurl.de:778/some/resource => https://www.myurl.de:778
+   * @param aUrl the url
+   * @return the base of the url
+   */
+  public static String getBaseUrl(final String aUrl) {
+    if (aUrl != null) {
+      int index = aUrl.indexOf("//");
+      if (index > 0) {
+        index = aUrl.indexOf("/", index+2);
+      } else {
+        index = aUrl.indexOf("/");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Put single-quotes around '/' to use the faster "indexOf(char)" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3027)